python/repositories.bzl

```diff
@@ -224,8 +235,6 @@ def python_register_toolchains(name, python_version, **kwargs):
 
     for platform in PLATFORMS.keys():
         sha256 = TOOL_VERSIONS[python_version]["sha256"].get(platform, None)
-        if not sha256:
```

Comment:
Also need to revert this.

Reply:
I don't think it should be reverted. `python_repository` should allow `sha256` to be optional. Which is common behavior of the `http_archive` and `http_file` rules.

Reply:
If anything I could see this being updated to a `fail` message but this skip leads to a confusing user experience IMO.

Reply:
Without this, it will break on versions that don't have a specific platform available.